egs/wsj/s5/utils/lang/limit_arpa_unk_history.py

```diff
@@ -0,0 +1,131 @@
+#!/usr/bin/env python3
+
+# Copyright 2018    Armin Oliya
+# Apache 2.0.
+
+# This script takes an existing ARPA lanugage model and limits the <unk> history 
+# to make it suitable for downstream <unk> modeling. This is for the case when
+# you don't have access to the original text corpus that is used for creating the LM.
+# If you do, you can use pocolm with the option --limit-unk-history=true. This
+# keeps the graph compact after adding the unk model.
+
+import argparse
+import io 
+import os
+import re 
+import sys 
+import gzip 
+from collections import defaultdict
+
+
+parser = argparse.ArgumentParser(
+	description = "This script takes an existing ARPA lanugage model and limits the <unk> history to make it suitable for downstream <unk> modeling.\nIt supports up to 5-grams.", 
+	usage = "utils/lang/limit_arpa_unk_history.py <oov_dict_entry> <input_arpa >output_arpa")
+
+parser.add_argument('oov_dict_entry', help='oov identifier, for example <unk>', type=str)
```

Comment:
I think for stylistic consistency I'd prefer to have this be 'oov-dict-entry'.  (The variable itself will still be oov_dict_entry).
And please have an example line, I think you can use the "epilog" for that.
For example:
`  epilog="E.g.: gunzip -c src.arpa.gz | utils/lang/limit_arpa_unk_history.py '<unk>' | gzip -c >dest.arpa.gz"`